.github/workflows/release.yml

```diff
@@ -81,6 +85,51 @@ jobs:
           if [ $PROJECT_NAME == "taichi-nightly" ]; then export PYPI_PWD="$NIGHT_PWD" && python build.py upload --skip_build --testpypi --project_name $PROJECT_NAME
           elif [ $PROJECT_NAME == "taichi" ]; then export PYPI_PWD="$PROD_PWD" && python build.py upload --skip_build; fi
 
+  build_and_upload_m1:
+    name: Build and Upload (Apple M1)
+    needs: matrix_prep
+    strategy:
+      matrix: ${{ fromJson(needs.matrix_prep.outputs.matrix_osx) }}
+    runs-on: [self-hosted, m1]
+    defaults:
+      run:
+        shell: "/usr/bin/arch -arch arm64e /bin/bash --noprofile --norc -eo pipefail {0}"
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+
+      - name: Build
+        run: |
+          export PATH=/Users/github/miniforge3/envs/$PYTHON/bin:$PATH
```

Comment:
Is this how conda is enabled 😅  ?

Reply:
Well... This is complicated... We can use conda activate $PYTHON. However, this won't change the PATH to python for some reason. Even if we activate conda env, we still need to manually set python path... So we can just get rid of the activate.

Reply:
BTW, the conda related github actions all requires setting PATH